src/coffee/mappings.coffee

```diff
@@ -0,0 +1,100 @@
+_ = require 'underscore'
+_.mixin require('underscore-mixins')
+csv = require 'csv'
+CONS = require './constants'
+
+class CustomFieldMappings
+
+  constructor: (options = {}) ->
+    @errors = []
+
+  mapFieldTypes: (fieldDefinitions, typeDefinitionKey, rowIndex, key, value,langHeader) ->
```

Comment:
Nitpick: usually if we start having more then 2 arguments, I would suggest you use named arguments, makes it easy to refactor things.

```coffeescript
mapFieldTypes: ({ fieldDefinitions, typeDefinitionKey, rowIndex, key, value, langHeader }) ->
```